src/mono/mono/component/hot_reload.c

```diff
@@ -1668,7 +1668,15 @@ apply_enclog_pass1 (MonoImage *image_base, MonoImage *image_dmeta, DeltaInfo *de
 				i++; /* skip the next record */
 				continue;
 			}
-			/* fallthru */
+			// don't expect to see any other func codes with this table
+			g_assert (func_code == ENC_FUNC_DEFAULT);
+			if (!is_addition) {
+				// Roslyn sometimes sends this when a custom attribute is deleted from
+				// a type definition.
+				break;
+			}
```

Comment:
This seems unnecessary.

Reply:
Yea there was more code here initially, but I wittled it down to just a comment ;-)